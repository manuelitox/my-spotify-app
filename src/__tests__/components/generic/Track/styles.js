import React from 'react'
import { shallow } from 'enzyme'

import StyledTrack from 'components/generic/Track/styles'

describe('Component | StyledTrack', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledTrack />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})