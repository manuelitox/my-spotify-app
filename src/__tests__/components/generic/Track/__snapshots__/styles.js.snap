// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledTrack should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  color: ",
          [Function],
          ";
  cursor: pointer;
  display: grid; 
  grid-template-columns: 40px 1fr;
  grid-gap: 20px;
  h4Â {
    font-size: 1.4rem;
    font-weight: 500;
    margin: 2px 0 5px;
  }
  p {
    color: ",
          [Function],
          ";
    font-size: 1.2rem;
    margin: 0;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "isPopUpVersion": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
